src/Infrastructure.EntityFramework/Repositories/CollectionRepository.cs

```diff
@@ -427,7 +484,64 @@ public async Task<ICollection<CollectionDetails>> GetManyByUserIdAsync(Guid user
         {
             var dbContext = GetDatabaseContext(scope);
             var query = CollectionAdminDetailsQuery.ByCollectionId(collectionId, userId).Run(dbContext);
-            var collectionDetails = await query.FirstOrDefaultAsync();
+
+            CollectionAdminDetails collectionDetails;
+
+            // SQLite does not support the GROUP BY clause
```

Comment:
Boo :( If this is the case, should we just have the single implementation? I guess the Sqlite version here is arguably less performant because it doesn't use delayed execution. But in practice I'm not sure it would make a significant difference, and it avoids having divergent EF implementations for different databases. Would be interested in what @justindbaur thinks.

Reply:
I'm not too attached either way (a single implementation or split). We do have a few other repository calls here that have similar behavior, so its not entirely new, if still pretty ugly. 

I'm also not too familiar with the magnitude of the performance difference, but I do expect this to be called fairly frequently in the AC until we have better caching there, so its something to keep in mind.